libtvdb/__init__.py

```diff
@@ -208,20 +208,28 @@ def get_paged(self, url_path: str, *, timeout: float) -> List[Any]:
 
         return all_results
 
-    def search_show(self, show_name: str, *, timeout: float = 10.0) -> List[Show]:
+    def search_show(self, show_name: str = None, imdb_name: str = None, *, timeout: float = 10.0) -> List[Show]:
```

Comment:
Since `show_name` and `imdb_name` can easily be mixed up now, the `*` should be moved to just after `self`.